### PR TITLE
fixes an oversight in plastic flap logic that got introduced when crawling was added a few years ago

### PR DESCRIPTION
**Pull request by alexkar598**
Closes #11037

# General Documentation

### Intent of your Pull Request
You can no longer crawl under plastic flaps, dead people can still pass through, it changes the requirement from being standing down to being standing down and not being able to move so stunned people can still be thrown through but alive people can't.

### Why is this change good for the game?
Nerfs powergamers and greytiders who break into the less secure flaps entrances. There's a reason it takes 50 years to deconstruct them, they're meant to be somewhat secure.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Plastic flaps no longer allow you to pass through by crawling

# Changelog

:cl:  
bugfix: You can no longer crawl under plastic flaps.
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
ANGERY GREYTIDER NOISES

**Comment by ynot01** [comment]
image

**Comment by alexkar598** [comment]
> downvoting a fix

**Comment by UselessTheremin** [comment]
There’s... literally windoors to prevent this

**Comment by alexkar598** [comment]
windoors you can break with a toolbox easily in 20 seconds

**Comment by alexkar598** [comment]
And for your infomation, the windoors are there so people can't ride a mulebot to get in without leaving a trace

**Comment by UselessTheremin** [comment]
> windoors you can break with a toolbox easily in 20 seconds

Doors you can hack easily in less than 20 seconds. Plus, remember when windoors got buffed? Yeah it takes about a minute to break now

**Comment by athena413** [comment]
i think this is good because it lets us put more plastic flaps in secure areas as well as we can stop considering "plastic flaps = big security risk"

at the same time, it is kind of an expected feature that plastic flaps are able to be crawled under, plus this doesnt even really fix the "bug" properly. either make players FULLY unable to go through flaps even when stunned and thrown through or keep flaps crawlable

regardless, this isnt a bugfix, it's a feature removal at this point. even if it was originally an oversight, it's basically just a feature for any codebase that has these flaps. stop pretending its a bugfix

**Comment by alao3alao3** [comment]
this is downright feature removal at this point, everyone crawls under flaps

**Comment by ToasterBiome** [comment]
Never saw someone that wasn't in that department use their flaps. Also this makes expected interactions with the flaps really confusing. If this gets merged expect exactly 0 players to ever interact with the flaps again, just mapchange and remove them at that point.

**Comment by alexkar598** [comment]
Flaps were introduced specifically to allow mulebots to deliver crates without making a massive hole that allows anyone to break in. Allowing crawling made them a massive hole that allows anyone to break in.

**Comment by SomeguyManperson** [comment]
Tfw labeling a balance change a "fix"

**Comment by alexkar598** [comment]
It is a fix to an oversight, crawling wasn't added to allow people to break in departments via flaps, this fixes that issue.

**Comment by alexkar598** [comment]
Also
> at the same time, it is kind of an expected feature that plastic flaps are able to be crawled under, plus this doesnt even really fix the "bug" properly. either make players FULLY unable to go through flaps even when stunned and thrown through or keep flaps crawlable

No, not really, crawling in itself is a weird unexplained mechanic, most players will encounter it by slipping on a wet floor and trying to move. The first and last time they will ever use rest to crawl intentionally is going to by watching someone rest under plastic flaps and moving through. There's nothing intuitive about resting then moving, this isnt a shooter where you go prone then crawling or w/e. Rest says it in the name its for "resting" ie a roleplay thing or whatever. Its not labelled "crawl" and most players beyond the ones that are part of the meta or w/e will never think of resting under flaps until they see someone else do it.

**Comment by SomeguyManperson** [comment]
Reinforced windows were added around each plastic flap that enters a depaetment specifically to fix that though and those take like a year to break even with tools

**Comment by patpol4** [comment]
Counterintuitive, if you want to make balance changes here maybe give the crawl under flaps a long timer?

**Comment by maxion12345** [comment]
NOOOOOOOOOOOOOOOOOOOOOOOOO YOU CANT CRAWL UNDER THE FUCKING FLAPS????????????????????? THEY ARE ONLY TO LET CRATES AND ROBOTS AND BOXES AND NON-LIVING THINGS THROUGH!!!!!!!!!!!!!!!!!!!!! YOUR JUST FUCKING GREYTIDING OH MY FUCKING GODDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD

also windoors exist. and most department delivery areas are in maint so you'll have to break in with tools anyways.

**Comment by Cartlord** [comment]
I have personally had issues with people tiding through windoors by just beating them down, and I do agree that it's rather difficult to actually find out you CAN crawl under plastic flaps, but either make them impenetrable or make them something you can walk through - this only-if-stunned crap is just WEIRD.

**Comment by MegaEmpirical** [comment]
Crawling in general introduced several issues that been overlooked, I don't mind being able to crawl under flaps if other changes are made to compensate but currently it's too easy to crawl under them and then break the windoor.
